src/PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp

```diff
@@ -3,37 +3,54 @@
 
 #pragma once
 
+#include <cstddef>
+#include <memory>
 #include <string>
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
-#include "DataStructures/Tensor/TypeAliases.hpp"
 #include "PointwiseFunctions/Elasticity/ConstitutiveRelations/ConstitutiveRelation.hpp"
+#include "Utilities/TMPL.hpp"
 
-/// \cond
 namespace Elasticity {
 namespace Tags {
-template <size_t Dim>
-struct Stress;
-}  // namespace Tags
-}  // namespace Elasticity
-namespace Tags {
-template <size_t Dim, typename Frame>
-struct Coordinates;
-}  // namespace Tags
-/// \endcond
-
-namespace Elasticity {
-namespace Tags {
-
-/// Base tag for the constitutive relation
-struct ConstitutiveRelationBase : db::BaseTag {};
 
-template <typename ConstitutiveRelationType>
-struct ConstitutiveRelation : ConstitutiveRelationBase, db::SimpleTag {
-  static constexpr OptionString help = {
-      "The constitutive relation of the elastic material"};
-  using type = ConstitutiveRelationType;
+/*!
+ * \brief The elastic material's constitutive relation.
+ *
+ * \see `Elasticity::ConstitutiveRelations::ConstitutiveRelation`
+ */
+template <size_t Dim>
+struct ConstitutiveRelation : db::SimpleTag {
   static std::string name() noexcept { return "Material"; }
+  using type =
+      std::unique_ptr<ConstitutiveRelations::ConstitutiveRelation<Dim>>;
+};
+
+/*!
+ * \brief The elastic material's constitutive relation, loaded from options in
+ * the `ProviderOptionTag`.
+ *
+ * Retrieves the constitutive relation from the object constructed from the
+ * `ProviderOptionTag` by calling its `constitutive_relation()` member function.
+ * Also requires the `ProviderOptionTag::type` to provide a `volume_dim` static
+ * variable and a `constitutive_relation_type` type alias.
+ *
+ * The constitutive relation can be retrieved from the DataBox using its base
+ * `Elasticity::Tags::ConstitutiveRelation` tag.
+ */
```

Comment:
This is my understanding of our classification: Parallel components and actions can have `initialization_tags` which will be inserted into the initial DataBox, and which will be removed by the `RemoveOptionsAndTerminatePhase` action. They must be option-creatable, because there's no other way to insert them into the DataBox at that point. But also the `const_global_cache_tags` are created from options, and that's where this constitutive relation tag is intended to live (but doesn't have tp). So I think the classification is that all `initialization_tags` and `const_global_cache_tags` must be "option-creatable". I actually intend to add an `OptionCreatableTag` protocol (once #1769 is merged) to make this requirement clear.
I think the documentation here makes clear that the tag is option-creatable, but let me know if you agree with my understanding.

Reply:
Ah, understood. Yes, this is clearly option-creatable, so that is fine.